packages/core/src/builder.class.ts

```diff
@@ -1513,10 +1500,8 @@ export class Builder {
     return this;
   }
 
-  get previewingModel() {
-    const search = this.getLocation().search;
-    const params = QueryString.parse((search || '').substr(1));
-    return params['builder.preview'];
+  get previewingModel(): string | undefined {
+    return Builder.isBrowser ? QueryString.parse((this.getLocation().search || '').substr(1))['builder.preview'] : undefined;
```

Comment:
is this change necessary for this PR? I'm concerned with some of these changes being breaking and seemingly out of scope for this PR so could have slipped in silently and problematically

this change needs a revert, getLocation() supports server side _and_ client side, and those using server side rendering (e.g. next.js SSR, angular, etc) need the current behavior

Reply:
aka they expect a server side preview when applicable, or else they worry "why is the content flashing when I preview pages, I think I shouldn't publish this as there seems to be an issue"